extensions/2.0/Khronos/KHR_texture_basisu/README.md

```diff
@@ -0,0 +1,173 @@
+# KHR_texture_basisu
+
+## Contributors
+
+* Members of 3D Formats Working Group
+* Arseny Kapoulkine, [@zeuxcg](https://twitter.com/zeuxcg)
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+This extension adds the ability to specify textures using KTX v2 images with Basis Universal supercompression. An implementation of this extension can use such images as an alternative to the PNG or JPEG images available in glTF 2.0 for more efficient asset transmission and reducing GPU memory footprint. Furthermore, specifying mip map levels is possible.
+
+When the extension is used, it's allowed to use value `image/ktx2` for the `mimeType` property of images that are referenced by the `source` property of `KHR_texture_basisu` texture extension object.
```

Comment:
Would it be useful to add links for Basis Universal and image/ktx2 in case people don't know what they are?

For Basis Universal, maybe the [GitHub page](https://github.com/BinomialLLC/basis_universal)?
For image/ktx2, maybe [this link](https://www.iana.org/assignments/media-types/image/ktx2)?